.github/workflows/release-please.yml

```
@@ -42,16 +42,16 @@
-          #name: Slides
-          #path: |
-          #  slides/Snakemake_HPC_Creators.pdf;
-          #  slides/Snakemake_HPC_Users.pdf;
-          #  slides/Snakemake_HPC_User_Creator_Combi.pdf;
-          #  slides/Snakemake_HPC_Admins.pdf;
+      - name: Upload Release Artifacts
+        if: ${{ steps.release.outputs.release_created }}        
+        uses: actions/upload-artifact@v4
+        with:
+        name: Slides
+           path: |
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 50: mapping values are not allowed in this context

(syntax-check)